### PR TITLE
maybe typos?

### PR DESCRIPTION
**Pull request by zeldela**
possible locale name + ID typos in west virginia and virginia 

discovered when testing a function I made that joins national FIPS codes to the voteshield locale IDs, PR [here](https://github.com/Voteshield/da-tools/pull/3).

### CODE REVIEW

**Comment by zzolo** [comment]
The West Virginia parts of this definitely look like I just had some typos.  It also doesn't look like this changes the IDs, so it shouldn't affect data or processing.  Will have to update the frontend.

@zeldela I'm not familiar enough with Virginia's data to make a call on that one.

File: reggie/configs/primary_locale_names/county/virginia.json
Comment (cvcotton) [inline reggie/configs/primary_locale_names/county/virginia.json:192]: @zeldela - The string is actually "king & queen county" in the database, so the "id" field needs to remain "king & queen county". You can change the "name" field to whatever you think is best though (that's just the pretty-print name we want to send out in emails).

**Review by cvcotton — CHANGES_REQUESTED** [review]
Just change the virginia "id" field back, and then I can approve this.

FYI - To actually get this into production, you'll need to:
1. merge this branch once it's approved
2. create an Inspector PR that updates reggie to the most recent commit of reggie (with this change) and get that approved & merged
3. like Alan mentioned, create a frontend PR with the same changes here https://github.com/Voteshield/frontend/blob/main/src/data/counties/va.json and here https://github.com/Voteshield/frontend/blob/main/src/data/counties/wv.json

File: reggie/configs/primary_locale_names/county/virginia.json
Comment (zeldela) [reply reggie/configs/primary_locale_names/county/virginia.json]: got it -- changed this. does it need to be "&" in the virginia.yaml, too?

Comment (cvcotton) [reply reggie/configs/primary_locale_names/county/virginia.json]: the "county_names" in the yaml is never actually used in the code, so *shrug* it doesn't matter to me - But if you are using it to refer to database values then yes it should be "&"

**Review by cvcotton — APPROVED** [review]
ok cool!

**Pull request merged**
